src/CoreWCF.Http/tests/CoreWCF.Http.GeneratedOperationInvokers.Tests.csproj

```diff
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
-  <Import Project="CoreWCF.Http.Tests.csproj" />
+  <PropertyGroup>
+    <ImportDirectoryPackagesProps>false</ImportDirectoryPackagesProps>
+  </PropertyGroup>
+  <Import Project="CoreWCF.Http.Tests.proj" />
```

Comment:
I did not know that trick. Nice

Reply:
It's not needed now though as I moved the Http.Tests.csproj bulk of work into a proj file. The problem I was having was there's now a build warning about pulling in the SDK twice, which resulted in Directory.Packages.Props getting pulled in twice which was causing a large amount of noise on restore. Moving it to a proj file solved that so I'm going to remove this now.